### PR TITLE
Fixes the Newsfeed

### PR DESCRIPTION
**Pull request by nosyrbllewe**
After the recent newsfeed refactor, the entire newsfeed on the frontend broke. This PR fixes the newsfeed so that it works again.

### CODE REVIEW

File: fakebook/src/app/component/newsfeed/newsfeed.component.ts [hunk 1]
```diff
@@ -22,23 +22,26 @@ export class NewsfeedComponent implements OnInit {
...
         }
-      } else {
-        this.getPosts();
-      }
+      } 
     });
   }
 
-  getPosts(): void {
+  getPosts(loadUserFeed : boolean): void {
```
Comment (escalonn) [inline fakebook/src/app/component/newsfeed/newsfeed.component.ts:37]: what for? you're not even using the parameter.

File: fakebook/src/app/component/newsfeed/newsfeed.component.ts [hunk 2]
```diff
@@ -22,23 +22,26 @@ export class NewsfeedComponent implements OnInit {
...
     this.newsfeedService
-      .getPosts()
-      .subscribe((gotPosts) => (this.posts = gotPosts));
+      .getUser()
+      .subscribe((gotUser) => { 
+        this.user = gotUser;
+         this.newsfeedService.getPosts(gotUser?.followers!)
+         .subscribe((gotPosts) => (this.posts = gotPosts));
```
Comment (escalonn) [inline fakebook/src/app/component/newsfeed/newsfeed.component.ts]: this isn't good, so at least open a github issue about it -
example link on the subject
https://www.thinktecture.com/en/angular/rxjs-antipattern-1-nested-subs/

File: fakebook/src/app/component/newsfeed/newsfeed.component.ts (hunk 1 above)
Comment (nosyrbllewe) [reply fakebook/src/app/component/newsfeed/newsfeed.component.ts:37]: Oops, forgot to finish that in the rush of things. The method should now only load the user feed if it is true.

File: fakebook/src/app/component/newsfeed/newsfeed.component.ts (hunk 2 above)
Comment (nosyrbllewe) [reply fakebook/src/app/component/newsfeed/newsfeed.component.ts]: Created issue #90.  I will admit that it doesn't feel nice at all and that it definitely needs a better implementation.

**Review by escalonn — APPROVED** [review]

**Pull request merged**